src/passwd.c

```diff
@@ -838,6 +843,8 @@ int main (int argc, char **argv)
 				break;
 			case 'R': /* no-op, handled in process_root_flag () */
 				break;
+            case 'P': /* no-op, handled in process_prefix_flag () */
```

Comment:
(please do use the same tabbing as the lines around it)

Reply:
Done, thanks.